### DISCUSSION TITLE
Stuck in "Applying elasticsearch Salt state"

### DISCUSSION THREAD

**Discussion by rusqq** [Security-Onion-Solutions/securityonion#7110] [Unsupported Versions]
Hi,

My search node installation stucks on "Applying elasticsearch Salt state". I can wait for almost an hour but it stucks in 64%. I see it's a common problem, but I don't see a resolve.

OS: Ubuntu 20.04 on VMware
vCPU: 4
RAM: 12 GB
HDD: 270GB

**Comment by rusqq** [comment]
I tried for example to change a hostname, someone told it worked, but not for me: https://groups.google.com/g/security-onion/c/icjMHlPhg_8

**Comment by rusqq** [comment]
hi, anyone?

**Comment by dougburks** [comment]
Please provide more information:

What options did you choose in Setup?

Are you using DHCP or a static IP address?

How many virtual NICs do you have and how are they configured?

Have you checked the Setup log in `/root/sosetup.log` for additional clues?

**Reply by rusqq** [reply]
Hi @dougburks ,

1. I'm using static IP
2. I'm using one virtual NIC, it's configured in the same subnet as manager and forward node.
3. /root/sosetup.log:

>                   Exiting: couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at https://HOSTNAME:9200: Get "https://HOSTNAME:9200": dial tcp IP_SEARCH_NODE:9200: connect: connection refused

In addition, so-status says curator, filebeat, nginx, telegraf, sensoroni and docker are installed correctly. There is an error in so-logstash, and so-elasticsearch is missing.

I don't use any stateful firewalls

**Reply by dougburks** [reply]
Check DNS and hostname resolution to ensure that the node can resolve its own hostname to its own IP address. Also make sure that there are no hostname or IP address conflicts on your network.

Then check to see if there are any Elasticsearch logs at `/opt/so/log/elasticsearch/` that might provide additional clues.

Also check the output of:
```
sudo docker logs so-elasticsearch
```

**Comment by rusqq** [comment]
any update please??
